brambling/forms/organizer.py

```diff
@@ -265,6 +266,30 @@ def _post_clean(self):
             self._update_errors(e)
 
 
+class CustomFormForm(forms.ModelForm):
+    class Meta:
+        model = CustomForm
+        exclude = ('event',)
```

Comment:
I say this because it confused me and I had the advantage of having already looked over the code.